### PR TITLE
Fix bugs

### LINKED ISSUE #124: [Add] Inconsistency in command format
Most commands follow a <letter>/ <Data> format such as t/ friends. However, add has an inconsistency shown below. Where some use acronyms and other uses full words and after the /.
add /name John Doe /class 4H /phone 98765432 t/friends t/owesMoney

### LINKED ISSUE #130: [help] Outdated list of commands
Need to update help

### PR DESCRIPTION
**Pull request by yimjianbing**
Fixes #130 , Fixes #124

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -6,9 +6,9 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("/name ");
-    public static final Prefix PREFIX_CLASS = new Prefix("/class ");
-    public static final Prefix PREFIX_PHONE = new Prefix("/phone ");
+    public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_CLASS = new Prefix("c/");
+    public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
```
Comment (wenjebs) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:12]: lgtm

File: src/main/java/seedu/address/logic/parser/CliSyntax.java
Comment (hyxnnii) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:1]: Good standardisation of command prefixes

File: src/main/resources/images/gb.png
Comment (hyxnnii) [inline src/main/resources/images/gb.png:1]: I like the colour!

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -18,12 +18,19 @@ public class HelpWindow extends UiPart<Stage> {
...
             + "7. help\n"
-            + "8. list\n";
+            + "8. list\n"
+            + "9. clear\n"
+            + "10. group\n"
+            + "11. groups\n"
+            + "12. findGroup\n"
+            + "13. deleteGroup\n"
+            + "14. export\n"
+            + "15. import\n";
```
Comment (Hoodini231) [inline src/main/java/seedu/address/ui/HelpWindow.java:33]: Should we put these in alphabetical order?

File: src/main/java/seedu/address/ui/GroupCard.java [hunk 3]
```diff
@@ -31,16 +31,19 @@ public class GroupCard extends UiPart<Region> {
...
     /**
-     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     * Creates a {@code GroupCode} with the given {@code Group} and index to display.
      */
     public GroupCard(Group group, int displayedIndex) {
         super(FXML);
         this.group = group;
         id.setText(displayedIndex + ". ");
         name.setText(group.getGroupName().toString());
+        studentName.setText("Students: ");
```
Comment (Hoodini231) [inline src/main/java/seedu/address/ui/GroupCard.java:46]: Do we need trailing whitespace after Students: ?

**Review by Hoodini231 — APPROVED** [review]
Looks good, I commented some questions about the code and if you feel its unneeded change lets merge!

**Pull request merged**